bin/critterget.py

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/python
-import urllib2
-import urllib
+import datetime
 import json
 import sys
-import datetime, base64
-import splunk.entity as entity
+import requests
+
+try:
+    import splunk.entity as entity
+except ImportError:
+    import splunk_mock.entity as entity
```

Comment:
This seemed like the simplest solution, and mimics patterns I've seen elsewhere in our code base.